xtractmime/_patterns.py

```diff
@@ -38,23 +40,28 @@
         b"MThd\x00\x00\x00\x06",
         b"\xff\xff\xff\xff\xff\xff\xff\xff",
         "audio/midi",
-    ),  # The string "MThd" followed by four bytes representing the number 6 in 32 bits (big-endian), the MIDI signature
+    ),  # The string "MThd" followed by four bytes representing the number 6 in 32 bits
+    # (big-endian), the MIDI signature
     (
         b"RIFF\x00\x00\x00\x00AVI ",
         b"\xff\xff\xff\xff\x00\x00\x00\x00\xff\xff\xff\xff",
         "video/avi",
-    ),  # The string "RIFF" followed by four bytes followed by the string "AVI ", the AVI signature
+    ),  # The string "RIFF" followed by four bytes followed by the string "AVI "
+    # the AVI signature
     (
         b"RIFF\x00\x00\x00\x00WAVE",
         b"\xff\xff\xff\xff\x00\x00\x00\x00\xff\xff\xff\xff",
         "audio/wave",
-    ),  # The string "RIFF" followed by four bytes followed by the string "WAVE", the WAVE signature
+    ),  # The string "RIFF" followed by four bytes followed by the string "WAVE"
+    # the WAVE signature
 )
 
 FONT_PATTERNS = (
     (
-        b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00LP",
-        b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff",
+        b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+        b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00LP",
+        b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+        b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff",
```

Comment:
I suggested using parentheses here for readability. Without them, you need to check whether or not there is a comma at the end, while with parentheses it’s more obvious. I believe black won’t remove the parentheses.